scripts/algorithms/l2-svm-predict.dml

```diff
@@ -25,6 +25,19 @@
 # Given ground truth labels, the script will compute an 
 # accuracy (%) for the predictions
 #
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME      TYPE        DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X         String      ---         Location to read the test feature matrix X
+# Y         String      ---         Location to read the true label matrix Y
```

Comment:
Yeah, let's just note in the description that this is optional.  We could either use @j143's suggestion, or modify it to `[OPTIONAL] Location to read the true label matrix Y.  Only needed for evaluating performance (accuracy, confusion) of the model.`.  I think the latter would be slightly easier for the user to quickly note that this is optional.

The alternative is to add a new "Optional?" (or "Required?") column.